### PR TITLE
Feature/eliminate health check 6892

### PR DESCRIPTION
**Pull request by netcriptus**
Fixes: #6892

In presence-less Raiden, health checks and whitelists are no longer necessary. These have been removed in previous tickets, that left some dead code behind.

This PR is removing dead code.

### CODE REVIEW

**Comment by istankovic** [comment]
@netcriptus, it seems "Fix a reference to pylintrc" is still here to confuse... could you please try rebasing?

File: raiden/raiden_event_handler.py [hunk 1]
```diff
@@ -287,7 +287,7 @@ def handle_send_refundtransfer(
         raiden: "RaidenService",
         refund_transfer_event: SendRefundTransfer,
         message_queues: Dict[QueueIdentifier, List[Tuple[Message, Optional[AddressMetadata]]]],
-    ) -> None:
+    ) -> None:  # pragma: no unittest
```
Comment (karlb) [inline raiden/raiden_event_handler.py]: Not sure anyone really cares for these pragmas anymore.

**Review by karlb — APPROVED** [review]
I'm always happy about removing code!

File: raiden/raiden_event_handler.py (hunk 1 above)
Comment (netcriptus) [reply raiden/raiden_event_handler.py]: ✅

**Pull request merged**